src/control/server/config.go

```diff
@@ -53,6 +53,13 @@ const (
 type networkProviderValidation func(string, string) error
 type networkNUMAValidation func(string, uint) error
 
+// Configuration information shared with the Agent
+type AgentCfg struct {
+	CRT_PHY_ADDR_STR   string
+	CRT_CTX_SHARE_ADDR uint32
+	CRT_TIMEOUT        uint32
```

Comment:
For the record, these field names are legal, but they don't follow convention, where [ALLCAP_VARS_ARE_FROWNED_UPON](https://golang.org/doc/effective_go.html#mixed-caps). I also suggest naming them for future control plane developers who would like to understand what they're for without having to sift through the C code to figure it out.

Reply:
Sure, I was just preserving the naming convention from the environment variable.  I've changed it.